openedx/core/djangoapps/programs/tests/test_views.py

```diff
@@ -1,6 +1,7 @@
 """
 Tests for the Programs.
 """
+import ddt
```

Comment:
Minor: ddt is a third-party module, so it should be in the group of imports below. Relatedly, unittest is part of the standard library; could you please move it here?